documentation/Architecture.md

```diff
@@ -2,7 +2,7 @@
 PRIME is a traditional N-Tier application architecture. It's composed of a frontend Angular application that communicates to a RESTful .Net Core API which persists it's data in a Postgres Database. We use Keycloak to authenticate users against BCSC and SiteMinder IDIR. There is one external integration with a HIBC REST API that validates College Licences against our users.
 
 ## Technologies in use
-- .Net Core 3.1 C# 
+- .Net Core 3.1 C#
```

Comment:
If we're stating the version of .net we should probably also state the version of c#

Reply:
Feels like this is out of scope we want to keep this MVP

Reply:
@joliesummers can you make a second ticket to add the language version to the docs ![image](https://user-images.githubusercontent.com/599498/100943612-6e0b7600-34b2-11eb-8cc2-593a5aada3f8.png)